File: JAVA/src/arrayEasy/largestElement.java
package arrayEasy;

import java.util.Arrays;

public class largestElement {

    public static void main(String[] args) {
        int[] arr = {5,3,2,1,4,6,8};
        int max = arr[0];
        System.out.println(Arrays.toString(arr));
        for (int j = 1;j < arr.length;j++) {
            if (arr[j] > max) {
                max = arr[j];
            }
        }
        System.out.println("Largest: "+max);
    }
}

File: PYTHON/01/arrays_easy_problems/largest_element.py
def function(arr):
    max = float('-inf')
    for i in range(len(arr)):
        if(arr[i] > max):
            max = arr[i]
    return max
arr = [2,4,6,3,9,1]
print(f"Max value from array({arr}) is: {function(arr)}")